Yoast/javascript
# Add a dismissable Alert component with 4 different styles: info, warning, error & success.

## Summary

Adds an Alert component to Yoast components. It has 4 different styles that can be set in its props: info, warning, error & success. It has an option to be dismissable, which will be saved in a cookie. The alert is based on this design: https://github.com/Yoast/design/issues/371#issuecomment-487953202

  * Package(s) involved:
  * Should the change be included in the package changelog?
  * Should the change be included in one or more plugin changelogs?
  * Package changelog item (if applicable): 
* Adds a dismissable Alert component with 4 different styles: info, warning, error & success.

## Relevant technical choices:

* Added the component to the example App under the section 'components'.
* Removed the svg files and replaced them with inline generated svg files in the alert component.
* Wrapped the App in <IntlProvider locale="en"> to make the component work, as it uses react-intl.
* Added grunt to @yoast/style-guide to be able to build the colors.json file.
* Added all the required colors to `_colors.scss`. It's a lot of new colors so it makes the file a bit messy. Would like a second opinion on how to make this cleaner. 

## Test instructions
This PR can be tested by following these steps:

* Checkout this branch. Go into the apps/components folder and run yarn install, then yarn start
* Open the website on the provided port and go into the components tab.
* Here you should see the following 4 alerts:
<img width="996" alt="Screenshot 2019-08-05 at 15 42 57" src="https://user-images.githubusercontent.com/16388834/62469763-3f379f80-b799-11e9-9e9d-1229b161b82f.png">
* Inspect them and make sure they match the specifications here: https://github.com/Yoast/design/issues/371#issuecomment-487953202
* Close them and make sure they don't come back unless you delete the cookies :)
* If you want to be thorough you can also yarn link to myYoast and test the alerts there.

## Impact check

## UI changes

## Quality assurance


Fixes #331
